src/main.c

```diff
@@ -6,7 +6,7 @@
 #include "err.h"
 #include "vm.h"
 
-static char *kernel_file = NULL, *initrd_file = NULL;
+static char *kernel_file = NULL, *initrd_file = NULL, *disk_file = NULL;
```

Comment:
For the sake of consistency, we can call it `diskimg_file`.

See https://en.wikipedia.org/wiki/Disk_image